src/main/webapp/i18n/de/result.json

```diff
@@ -10,7 +10,12 @@
             "updated": "Result aktualisiert mit ID {{ param }}",
             "deleted": "Result gelöscht mit ID {{ param }}",
             "delete": {
-                "question": "Soll Result {{ id }} wirklich dauerhaft gelöscht werden?"
+                "question": "Soll Result {{ id }} wirklich dauerhaft gelöscht werden?",
```

Comment:
I know you did not change anything there, but shouldn't it be rather called: Soll **Bewertung** {{ id }} wirklich dauerhaft gelöscht werden?

Reply:
I agree, and if so, then it should be changed in the whole file shouldn't it?

Reply:
True, I only looked at the changed translations. Seems like it is used all the time in this file. Can you adapt this @SimonEntholzer ? :)

Reply:
I changed it :)